index.js

```diff
@@ -51,6 +53,71 @@ app.post("/audio_info", upload.single("file"), async (req, res) => {
   }
 });
 
+app.post("/spotify-auth", async (req, res) => {
+  // console.log("request.body", req.body);
+  const payload = {
+    grant_type: "authorization_code",
+    code: req.body.code,
+    redirect_uri: process.env.REDIRECT_URI,
+  };
+  try {
+    const result = await axios({
+      method: "post",
+      url: "https://accounts.spotify.com/api/token",
+      data: qs.stringify(payload),
+      headers: {
+        Authorization: `Basic ${process.env.CLIENT_DETAILS}`,
+        "Content-Type": "application/x-www-form-urlencoded",
+      },
+    });
+    res.send(result.data);
+    console.log("result.data", result.data);
+  } catch (error) {
+    console.log(error);
```

Comment:
if there is an error I think we should be sending a 500 response, at the moment we are just logging

Reply:
Done